python/dwave_qbsolv/dimod_wrapper.py

```diff
@@ -31,7 +31,7 @@ def __init__(self):
     @dimod.decorators.bqm_index_labels
     def sample(self, bqm, num_repeats=50, seed=None, algorithm=None,
                verbosity=-1, timeout=2592000, solver_limit=None, solver=None,
-               target=None, find_max=False, **sample_kwargs):
+               target=None, find_max=False, n_solutions=20, **sample_kwargs):
```

Comment:
Can we rename this `num_reads`? I know `n_solutions` is consistent with QBSolv locally, but we use `num_reads` in all of the other Ocean samplers.